### PR TITLE
Add a test for a chrome bug related with draw buffers dirty state.

### PR DESCRIPTION
**Pull request by zhenyao**
See crbug.com/678153

This test fails without https://codereview.chromium.org/2611403002/, and passes with it.

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell @kainino0x Please review

File: sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+precision mediump float;
+
+out vec4 my_FragColor;
+void main() {
+    my_FragColor = vec4(1, 0, 0, 1);
+}
+</script>
+<script>
+"use strict";
+description("This test verifies a Chrome's draw buffers dirty state bug (crbug.com/678153).");
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html]: Minor grammar issue: "a bug in draw buffers dirty state management in Chrome".

File: sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  wtu.clearAndDrawUnitQuad(gl);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "Clear and draw should cause no GL errors");
+
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.DEPTH_ATTACHMENT, gl.TEXTURE_2D, null, 0);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, colorTex, 0);
+
+  wtu.checkCanvasRect(gl, 0, 0, width, height, [255, 0, 0, 255], "should be red");
+
+  gl.clearColor(0.0, 1.0, 0.0, 1.0);
+  gl.clear(gl.COLOR_BUFFER_BIT);
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html]: Could you add a comment here like "Previously in Chrome, switching around the attachments on a framebuffer affected a DrawBuffers cache that was maintained properly during draw calls, but not clears."?

File: sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html]: Done

File: sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html (hunk 2 above)
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/draw-buffers-dirty-state-bug.html]: Done

**Pull request merged**